### PR TITLE
Incoming SMS handling for netgear_lte

### PR DESCRIPTION
**Pull request by amelchio**
**Description:**

Add an event and a service call.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22402

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/netgear_lte.markdown
Comment (syssi) [inline source/_components/netgear_lte.markdown]: The inbox ID of the received message.

File: source/_components/netgear_lte.markdown
Comment (syssi) [inline source/_components/netgear_lte.markdown]: Same here.

File: source/_components/netgear_lte.markdown
Comment (amelchio) [reply source/_components/netgear_lte.markdown]: Thank you, fixed.

**Review by klaasnicolaas — APPROVED** [review]
✅ Can be merged after merging the parent PR 👍

**Pull request merged**
